mllib/src/main/scala/org/apache/spark/ml/evaluation/MultilabelClassificationEvaluator.scala

```diff
@@ -63,18 +63,18 @@ class MultilabelClassificationEvaluator (override val uid: String)
 
   setDefault(metricName -> "f1Measure")
 
-  final val metricClass: DoubleParam = new DoubleParam(this, "metricClass",
+  final val metricLabel: DoubleParam = new DoubleParam(this, "metricLabel",
     "The class whose metric will be computed in precisionByLabel|recallByLabel|" +
       "f1MeasureByLabel. Must be >= 0. The default value is 0.",
     ParamValidators.gtEq(0.0))
 
   /** @group getParam */
-  def getMetricClass: Double = $(metricClass)
+  def getMetricLabel: Double = $(metricLabel)
```

Comment:
Just because my memory is terrible... was this a param you introduced for 3.0 recently? like this isn't a breaking change from what was released right?

Reply:
yes, they are introduced recently.
maybe since the master is not released to version 3.0, so this is not a breaking change?

Reply:
Yes, not a breaking change if this wasn't in 2.x